src/ansys/simai/core/data/models.py

```diff
@@ -38,6 +114,43 @@ class ModelConfiguration:
     simulation_volume: Dict[str, Any] = None
     project_id: str = None
 
+    @property
+    def domain_of_analysis(self) -> DomainOfAnalysis | None:
+        """The Domain of Analysis of the model configuration."""
+        if not self.simulation_volume:
+            return None
```

Comment:
I'm wondering if returning an empty `DomainOfAnalysis` object would be better than just `None`.

Reply:
resolving as the method `domain_of_analysis` is removed.